engine/apps/api/views/user.py

```diff
@@ -234,9 +234,27 @@ def schedules_with_oncall_users(self):
         """
         return get_oncall_users_for_multiple_schedules(self.request.user.organization.oncall_schedules.all())
 
+    def _get_is_currently_oncall_query_param(self) -> str:
+        return self.request.query_params.get("is_currently_oncall", "").lower()
+
+    def _is_currently_oncall_request(self) -> bool:
+        return self._get_is_currently_oncall_query_param() in ["true", "false"]
+
+    def _is_short_oncall_request(self) -> bool:
+        return self.request.query_params.get("short", "true").lower() == "false"
```

Comment:
This seems confusing..? `_is_short_oncall_request` will return `True` if the `short` param value is `"false"`?

Reply:
ahh good catch, thanks 👍